### DISCUSSION TITLE
Unexpected error display after setup firebase project

### DISCUSSION THREAD

**Discussion by MichelMan** [AntlerVC/firetable#1123] [Support / Q&A] [answered]
**Describe the bug**
I have tried to follow the recommanded setup in the documentation: https://docs.rowy.io/setup/install

But after the setup of my firebase app, when I tried to access to my project, I have the following error page (see above).

Could someone explain me why I got this error and help me solve it.

Thanks in advance!

**Expected behavior**
Access to my firestore data throught Rowy Dashboard.

**Screenshots**
![qkdjqdkjkqjf](https://user-images.githubusercontent.com/8861295/215296213-1aa69596-0c28-48c7-b38c-19972a7dc5dc.png)

**Comment by BeeBombshell** [comment, marked as answer]
Hey @MichelMan, can you check if the account you're logged in on the Firebase is the same as the one you're using to create a project on Rowy? Also, since this issue seems to be an auth-related issue, which sign-in method have you used?
